src/datasources/config-api/config-api.service.ts

```diff
@@ -51,15 +51,13 @@ export class ConfigApi implements IConfigApi {
     }
   }
 
-  clearChains(): Promise<void> {
+  async clearChains(): Promise<void> {
     const pattern = CacheRouter.getChainsCachePattern();
-
-    return Promise.all([
+    await Promise.all([
       this.cacheService.deleteByKey(CacheRouter.getChainsCacheKey()),
       this.cacheService.deleteByKeyPattern(pattern),
-    ]).then(() => {
-      return;
-    });
+    ]);
+    return;
```

Comment:
I don't think we need to `return` anymore.
```suggestion
```

There are too many instances of this to mark, so I'll just comment here.

Reply:
Ups you are right! 🤦 Automatic suggestion by the IDE which also removed the `.then()`

Reply:
[5ea015b](https://github.com/safe-global/safe-client-gateway-nest/pull/611/commits/5ea015b998adf08743fbadb397c815e8f74b6a3c)